assets/src/domain/eventEditor/ui/datetimes/datesList/actionsMenu/dateEntityMainMenuItem/CopyDate.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { omit } from 'ramda';
+import { __ } from '@wordpress/i18n';
+
+import { Copy } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
+import { useDatetimeContext } from '@edtrHooks/index';
+import { useDatetimeItem } from '@edtrServices/apollo/queries';
+import { useDatetimeMutator } from '@edtrServices/apollo/mutations';
+
+const CopyDate = (props) => {
+	const { id } = useDatetimeContext();
+	const datetime = useDatetimeItem({ id });
+	const { createEntity } = useDatetimeMutator();
+	const newDatetime = omit(
+		[
+			'dbId',
+			'id',
+			'isActive',
+			'isExpired',
+			'isSoldOut',
+			'isTrashed',
+			'isUpcoming',
+			'length',
+			'status',
+			'__typename',
+		],
+		datetime
+	);
+	const onClick = () => createEntity(newDatetime);
+
+	return <Copy {...props} entityType='datetime' onClick={onClick} />;
```

Comment:
Better to pass the title here `__('copy date')`

Reply:
done, thx